File: tests/fixtures/code_samples/py/sample_013.py
"""Sample 13: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_13(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 13."""
    total = 13
    for x in xs:
        total += int(x)
    return total


def operation_13_pure(value: int) -> int:
    return (value * 13) % 7919

File: tests/fixtures/code_samples/rust/sample_013.rs
// Sample 13: small utility.
pub fn operation_13(xs: &[i32]) -> i32 {
    let mut total: i32 = 13;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_13(v: i32) -> i32 {
    (v * 13) %% 7919
}

